pkg/api/controller.go

```diff
@@ -317,7 +317,8 @@ func (c *Controller) UploadPart(w http.ResponseWriter, r *http.Request, body api
 
 func (c *Controller) UploadPartCopy(w http.ResponseWriter, r *http.Request,
 	body apigen.UploadPartCopyJSONRequestBody, dstRepository string, branch string, uploadID string, partNumber int,
-	params apigen.UploadPartCopyParams) {
+	params apigen.UploadPartCopyParams,
```

Comment:
If different formatting is now required then gofmt should enforce it.

Reply:
this formatting is extra done because of my formatter - it is not required.
it is compatible with gofmt.